packages/host/package.json

```diff
@@ -29,11 +29,9 @@
   },
   "dependencies": {
     "@fontsource/inter": "^4.5.0",
-    "@openmsupply-client/catalogue": "^0.0.0",
```

Comment:
Not sure why these are removed

Reply:
Ah yes. They were the 'container' packages which were removed previously, I just had forgotten to remove them here. When I `yarn add`'d, I noticed as it bugged out